lib/ansible/modules/system/user.py

```diff
@@ -271,6 +270,7 @@ class User(object):
     platform = 'Generic'
     distribution = None
     SHADOWFILE = '/etc/shadow'
+    SHADOWFILE_INDEX = 7
```

Comment:
shadowfile_expire_index? .. just index seems ambiguous and we might need others in future

Reply:
Very good point.